rs/nervous_system/proto/protobuf_generator/src/lib.rs

```diff
@@ -46,13 +38,7 @@ pub fn generate_prost_files(proto_paths: ProtoPaths<'_>, out_dir: &Path) {
         .nervous_system
         .join("ic_nervous_system/pb/v1/nervous_system.proto");
 
-    // Most types should be Copy (currently, only Image is not Copy).
-    for type_name in copy_type_names {
```

Comment:
Why this change to the generator?  This would mean that all of our proto types are now copy, which is not necessarily desirable